docs/dictionary/control_st/repeat.lcdoc

```diff
@@ -25,124 +55,222 @@ repeat with tVarNum = 1 to 20
   put tPrefix  x into tVarName
   do "put empty into "  tVarNameend repeat
 
-Example:
--- Loops with using 0 are validrepeat with tVarNum = 0 to 5 put tVarNum  comma after tTextend repeatput tTexttText = 0,1,2,3,4,5,
-
-Example:
--- Loops can also make use of negative numbersrepeat with tVarNum = 3 down to -5 put tVarNum  comma after tTextend repeatput tTexttText = 3,2,1,0,-1,-2,-3,-4,-5,
-
 Parameters:
-loopForm (enum): The loopForm is one of the following forms:
-- "forever"
-- "until condition"
-- "while condition"
-- "[for] number times"
-- "with counter = startValue [to | down to] endValue [step increment]"
-- "for each chunkType labelVariable in container"
-- "for each element labelVariable in array"
-- "for each key labelVariable in array"
-statementList: One or more LiveCode statements, and can also include if, switch, try, or repeat control structures.
-condition (bool): 
-number, startValue, endValue, and increment: Numbers or expressions that evaluate to numbers.
-counter or labelVariable: A legal variable name.
-chunkType: One of byte, codeunit, codepoint, character (or char), token, trueword, word (or segment), item, sentence, paragraph or line.
-container: Any existing container.
-array (array): Any existing container that contains an array of values.
+loopForm (enum): 
```

Comment:
Sorry, could you remove this `(enum)` too please? That should make the markdown render correctly.